minecraft_servers/risingpixels/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "server_name": "risingpixels",
+  "nice_name": "RisingPixels",
+  "direct_ip": "play.risingpixels.net",
+  "server_wildcards": [
+    "%.risingpixels.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://risingpixels.net",
+    "web_shop": "https://store.risingpixels.net",
+    "web_support": "https://discord.gg/aHvRGg4hqb",
```

Comment:
Web_support cannot be Discord, either contact form or FAQ
If nonexistent, remove